src/pipelines/encore/encore-pipeline.ts

```diff
@@ -0,0 +1,343 @@
+import { Resolution } from '../../models/resolution';
+import { Pipeline } from '../pipeline';
+import AWSPipeline from '../aws/aws-pipeline';
+import { AWSPipelineConfiguration } from '../aws/aws-pipeline-configuration';
+import { EncorePipelineConfiguration } from './encore-pipeline-configuration';
+import { QualityAnalysisModel } from '../../models/quality-analysis-model';
+import { EncoreInstance } from '../../models/encoreInstance';
+import { EncoreJobs, EncoreJob } from '../../models/encoreJobs';
+import logger from '../../logger';
+import fs from 'fs';
+const ISOBoxer = require('codem-isoboxer');
+const { Readable } = require('stream'); //Typescript import library contains different functions. 
+//If someone knows how to achieve the same functionality in Typescript syntax let me know.
+const { finished } = require('stream/promises'); //Same as above.
+
+export function delay(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+export class EncorePipeline implements Pipeline {
+
+  configuration: EncorePipelineConfiguration;
+
+  constructor(configuration: EncorePipelineConfiguration) {
+    this.configuration = configuration;
+  }
+
+  async transcode(input: string, targetResolution: Resolution, targetBitrate: number, output: string, variables?: Record<string, string>): Promise<string> {
+    return "todo"
+  }
+
+  async analyzeQuality(reference: string, distorted: string, output: string, model: QualityAnalysisModel): Promise<string> {
+
+    const AWSConf: AWSPipelineConfiguration = {
+      inputBucket: "vmaf-files-incoming",
+      outputBucket: "vmaf-files",
+      mediaConvertRole: "",
+      mediaConvertSettings: "",
+      mediaConvertEndpoint: "",
+      ecsSubnet: "subnet-05d98882c13408e16",
+      ecsSecurityGroup: "sg-0e444b67a747bf739",
+      ecsContainerName: "easyvmaf-s3",
+      ecsCluster: "vmaf-runner",
+      ecsTaskDefinition: "easyvmaf-s3:3"
+    };
+
+    const awsPipe = new AWSPipeline(AWSConf);
+    const result: string = await awsPipe.analyzeQuality(reference, distorted, output, model);
+    return result
```

Comment:
Here you can just do `return await awsPipe.analyzeQuality(reference, distorted, output, model);`